src/com/sun/javatest/Script.java

```diff
@@ -208,6 +211,10 @@ public abstract class Script {
     private boolean jtrIfPassed =
             System.getProperty("javatest.script.jtrIfPassed", "true").equals("true");
 
+    /**
+     * tries to search for a statusTransformer, no status transformations will be done if the value is null
+     */
+    private static final StatusTransformer statusTransformer = searchStatusTransformer();
```

Comment:
Typically, `static final` fields are named in `UPPER_CASE`

```suggestion
    private static final StatusTransformer STATUS_TRANSFORMER = loadStatusTransformer();
```

Reply:
oh yeah.. forgot about that convention, will change :-)